tools/migration/batch_config.template

```diff
@@ -0,0 +1,16 @@
+{
+    "old_instance" : {
+        "url": "http://localhost:8080",
+        "user": "user",
+        "token": "tokentoken"
+    },
+    "new_instance" : {
+        "url": "http://localhost:8080",
+        "user": "user",
+        "token": "tokentoken"
+    },
+    "project_mappings": [
+        {"from": "a", "to": "b"},
```

Comment:
use simple json hashmaps, e.g {"a": "b", "c": "d"}?

Reply:
I actually did that on purpose to prevent any kind of confusion from where to where the migration is happening

Reply:
makes sense